dev-docs/bidders/incrementx.md

```diff
@@ -0,0 +1,21 @@
+---
+layout: bidder
+title: IncrementX
+description: Prebid IncrementX Bidder Adaptor
+pbjs: true
+biddercode: incrementx
+media_types: banner
+gdpr_supported: true
```

Comment:
If you claim to support `gdpr` please add your `gvl_id` .

Also your bid adapter doesn't seem to pass any consent information along